content/docs/working_at_the_turing/reasonable_adjustments.md

```diff
@@ -0,0 +1,24 @@
+---
+# Page title as it appears in the navigation menu
+title: "Reasonable Adjustments"
+# Adjust weight to reorder menu items (lower numbers appear first)
+weight: 4
+# Uncomment to hide nested pages in a collapsed menu
+# bookCollapseSection = true
+# Uncomment to hide this page from the navigation menu
+# bookHidden = false
+# Uncomment to exclude this page from the search
+# bookSearchExclude = true
+---
+
+# Reasonable Adjustments Policy
+
```

Comment:
```suggestion
# Reasonable Adjustments Policy

'Reasonable adjustments' refer to changes to workplaces procedures or practices that are put in place to avoid a situation where a person with disabilities is disadvantaged compared to one without.

```

Just adding a definition